sdk/workloads/Azure.ResourceManager.Workloads/tests/Tests/PhpWorkloadsCRUDTests.cs

```diff
@@ -0,0 +1,113 @@
+using System.Linq;
+using System.Reflection;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Workloads.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Workloads.Tests.Tests
+{
+    public class PhpWorkloadsCRUDTests : WorkloadsManagementTestBase
+    {
+        private const string TestResourceNamePrefix = "sdk-tests-dotNet-php-";
+        private readonly string BasePath = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+
+        public PhpWorkloadsCRUDTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        [OneTimeTearDown]
+        public void Cleanup()
+        {
+            CleanupResourceGroups();
+        }
+
+        [RecordedTest]
+        [Test]
+        public async Task TestPhpWorkloadsCRUDOperations()
+        {
+            try
+            {
+                Resources.SubscriptionResource subscription = await Client.GetDefaultSubscriptionAsync();
+                Resources.ResourceGroupResource rg = await CreateResourceGroup(
+                    subscription,
+                    TestResourceNamePrefix,
+                    Core.AzureLocation.EastUS);
+
+                var rgName = rg.Data.Name;
+
+                var resourceJson = File.ReadAllText(Path.Combine(BasePath, "TestData", "PhpWorkloads", "PutPhpWorkload.json"));
+                JsonElement json = JsonDocument.Parse(resourceJson).RootElement;
+
+                var resourceData = PhpWorkloadResourceData.DeserializePhpWorkloadResourceData(json);
+
+                resourceData.ManagedResourceGroupConfiguration = new ManagedRGConfiguration()
+                {
+                    Name = rgName + "-mrg"
+                };
+
+                resourceData.DatabaseProfile.ServerName = rgName.ToLower();
+                resourceData.CacheProfile.Name = rgName;
+
+                // Create
+                ArmOperation<PhpWorkloadResource> createdResource =
+                    await rg.GetPhpWorkloadResources().CreateOrUpdateAsync(
+                        WaitUntil.Completed,
+                        rgName,
+                        resourceData);
+
+                Assert.IsTrue(createdResource.Value.Data.ProvisioningState ==
+                    Models.PhpWorkloadProvisioningState.Succeeded);
+
+                Assert.AreEqual(rgName, createdResource.Value.Data.Name);
+
+                // Read
+                Response<PhpWorkloadResource> getResource = await rg.GetPhpWorkloadResourceAsync(rgName);
+                Assert.IsNotNull(getResource);
+                Assert.AreEqual(rgName, getResource.Value.Data.Name);
+
+                // Update
+                var patchJson = File.ReadAllText(Path.Combine(BasePath, "TestData", "PhpWorkloads", "PatchPhpWorkload.json"));
+                PhpWorkloadResourcePatch updatePayload = JsonConvert.DeserializeObject<PhpWorkloadResourcePatch>(patchJson);
```

Comment:
Same as above, I hope we can use `JsonDocument.Parse` here.

Reply:
I tried using `JsonDocument.Parse` but I could not figure out how to convert it to an object of `PhpWorkloadResourcePatch` as there is no method that takes in `JsonElement` and convert it to that object. Hence I used `JsonConvert.DeserializeObject`. Let me know if you know of a way to do that!